core/arch/x86_64/kernel/thread_private.h

```diff
@@ -43,6 +43,7 @@ struct thread_ctx {
 	enum thread_state state;
 	vaddr_t stack_va_end;
 	vaddr_t stack_va_curr[MAX_TA_IDX];
+	vaddr_t thread_abt_stack_va_end;
```

Comment:
Can we remove original abt_stack_va_end?

Reply:
Can we still use abt_stack_va_end? and add comment "moved from thread_core_local" ?